string command
int64 x
int64 y
int64 z
int64 t
time sent_time
---
string responce
